docs/tables/aws_route53_vpc_association_authorization.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Steampipe Table: aws_route53_vpc_association_authorization - Query AWS Route53 for other-account VPCs."
+description: "Gets a list of the VPCs that were created by other accounts and that can be associated with a specified hosted zone because you've submitted one or more `CreateVPCAssociationAuthorization` requests.."
+---
+
+# Table: aws_route53_vpc_association_authorization - Query AWS Route53 for other-account VPCs using SQL
+
+Amazon Route 53 is a highly available and scalable Domain Name System (DNS) web service.
+
+## Table Usage Guide
+
+The `aws_route53_vpc_association_authorization` table in Steampipe provides you with information VPCs in other AWS accounts that are authorized to be associated with a specified `hosted_zone_id`.
+
+## Examples
+
+### Basic info
+Check which other-account VPCs are authorized
+
+```sql+postgres
+select
+  hosted_zone_id,
+  vpc_id,
+  vpc_region
+from
+  aws_route53_vpc_association_authorization
+where
+  hosted_zone_id = 'XXXXXXXXXXXXXXXXXXXXX';
```

Comment:
Instead of using `hosted_zone_id = 'XXXXXXXXXXXXXXXXXXXXX'`, could we use a dummy value for hosted_zone_id?

Reply:
Used example from https://docs.aws.amazon.com/cli/latest/reference/route53/list-hosted-zones.html#examples